megamek/src/megamek/server/Server.java

```diff
@@ -11335,18 +11340,21 @@ public void deployTeleMissile(Entity ae, WeaponType wtype, AmmoType atype, int w
                     vPhaseReport.add(r);
                     te.heatFromExternal += 2 * missiles;
                     Report.addNewline(vPhaseReport);
-                } else if (te instanceof Tank) {
-                    boolean targetIsSupportVee = (te instanceof SupportTank)
-                                                 || (te instanceof LargeSupportTank)
-                                                 || (te instanceof SupportVTOL);
+                } else if (te instanceof GunEmplacement){
```

Comment:
This was originally at the end in a catch-all `else` block, which was unreachable for gun emplacements, because they would match (te instanceof Tank). So I moved it ahead of the Tank block.